src/rely/reporters/JunitReporter.re

```diff
@@ -0,0 +1,63 @@
+let testResultToTestCase = (testResult: Reporter.testResult) => {
+  let time =
+    switch (testResult.duration) {
+    | Some(Seconds(s)) => s
+    | None => 0.
+    };
+
+  switch (testResult.testStatus) {
+  | Passed =>
+    Junit.Testcase.pass(
+      ~name=testResult.title,
+      ~classname=testResult.fullName,
+      ~time,
+    )
+  | Skipped =>
+    Junit.Testcase.skipped(
+      ~name=testResult.title,
+      ~classname=testResult.fullName,
+      ~time,
+    )
+  | Failed(description, _location_opt, message) =>
+    Junit.Testcase.failure(
+      ~message,
+      ~typ="???",
+      ~name=testResult.title,
+      ~classname=testResult.fullName,
+      ~time,
+      description,
+    )
+  | Exception(_e, _location_opt, message) =>
+    Junit.Testcase.error(
+      ~message,
+      ~typ="???",
```

Comment:
probably "Exception"

Reply:
From reading the docs it seems like it should be the error as a string